### DISCUSSION TITLE
Govt Github?  Très Bien!

### DISCUSSION THREAD

**Discussion by oenu** [canada-ca/welcome#59] [General]
This is so cool! Are there any other similar Canadian repos?
****
C'est trop cool! Existe-t-il d'autres pensions canadiennes similaires ?

**Comment by boydjaimie** [comment]
Here’s a good one!
https://github.com/bcgov

**Reply by oenu** [reply]
Awesome thank you!

**Comment by nschonni** [comment]
You can find some on here https://government.github.com/community/ and also https://code.open.canda.ca

**Comment by michaelpetryk** [comment]
I am very new to GitHub, sorry for my ignorance on this topic.  I am an employee of the Government of Canada that would like to publish their code on this site or any other appropriate Government of Canada repository.  How would I go about doing this?  michael.petryk@ecn.forces.gc.ca  I have permission from management to publish the code under Open Government Licence - Canada

**Reply by oenu** [reply]
Hey Micheal, I would strongly suggest reaching out directly to one of the maintainers, [listed here](https://github.com/orgs/canada-ca/people). If you click through to their profiles you can see what organizations they are a part of and contact information. Im not sure how Govt internal communications get verified but im sure emailing their govt account from your govt account would be the best in terms of authentication. 

@nschonni @'ing you so you see this

**Reply by nschonni** [reply]
@michaelpetryk you can see the guidance on publishing OSS on https://www.canada.ca/en/government/system/digital-government/digital-government-innovations/open-source-software.html. This organization is mostly used around things that are related to the Canada.ca site and TBS policies that are in flight to be published there, so isn't likely a good candidate for DND specific work. EX: DRDC has their own org over https://github.com/DND-DRDC-RDDC/.

PS: The Open Government License isn't really suitable for source code, as it's more targeted at data.
